### PR TITLE
Recipe App initial version 

### PR DESCRIPTION
**Pull request by TamaraDimov**
In this project @Salimer and I made the following :

Created the login page and registration page:
Built with Devise.
Food list:
Display a list of food as in the wireframe.
Button that leads to the form that allows users to add new food.
Recipes list:
Display a list of recipes created by the logged-in user as in the wireframe.
Button that leads to recipe details.
Current user is allowed to delete it own recipe.
Public recipe list:
-display a list of all public recipes ordered by newest as in the wireframe.
-create button that lead to recipe details.
General shopping list view
-shows the list of food that is missing for all recipes of the logged-in user.
count the total food items and total price of the missing food.
Make sure there are no N+1 queries happening.
Create a navigation menu that allows users to open all of the pages you created.

### CODE REVIEW

File: app/views/recipes/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        <div class="slider-container">
+        <span class="switch-label">Public</span>
+        <label class="switch">
+          <%= f.check_box :is_public, { class: "toggle-switch" }, 0, 1 %>
+          <span class="slider round"></span>
+        </label>
+      </div>
+      <%= f.hidden_field :is_public, value: @recipe.is_public ? 0 : 1 %>
+      <%= f.submit 'Toggle', class: 'hidden-submit', style: 'display: none;' %>
+    <% end %>
```
Comment (thecodechaser) [inline app/views/recipes/show.html.erb:19]: - [ ] Great job on the project requirements but there's one small issue. On the recipe details page if I mark any recipe public or private I can see the toggle button changes but when I go to the public recipe page I can't see the changes there Also when I come back to the recipe details page again, I can only see recipe old state(public or private). Please make sure to fix this issue and changes to the recipe public state should reflect and users should be able to see the changes on the public recipe page.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @TamaraDimov, and @Salimer👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight 🎉
- All tests are passed. ✔️
- Highly organized code. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salimer** [comment]
Thank you @thecodechaser for the thorough review, we addressed the required changes 🥳

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @TamaraDimov & @Salimer, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Ruby on Rails capstone project done successfully ✔️
- Good use of Git flow ✔️
- UI design and functionality ✔️ 

## Optional changes

- [x] You can kindly work more on the visual design of the app in order to have a wonderful nad great app.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**